src/components/FlowRunTimelineSelectionPanel.vue

```diff
@@ -71,8 +99,9 @@
 .task-run-panel { @apply
   border
   dark:border-background-600
-  bg-background-800
-  dark:bg-background
+  bg-background
+  bg-opacity-80
+  backdrop-blur
```

Comment:
Why the blur and change in opacity? Isn't this panel still part of the normal layout?

Reply:
In full screen it overlays the content:

<img width="813" alt="image" src="https://user-images.githubusercontent.com/6776415/226620980-a1cb8020-afee-4d9b-8d6d-e16d7180831a.png">